src/cacheHttpClient.ts

```diff
@@ -253,33 +264,35 @@ async function uploadFile(
 }
 
 async function commitCache(
-    restClient: RestClient,
+    httpClient: HttpClient,
     cacheId: number,
     filesize: number
-): Promise<IRestResponse<void>> {
-    const requestOptions = getRequestOptions();
+): Promise<ITypedResponse<any>> {
     const commitCacheRequest: CommitCacheRequest = { size: filesize };
-    return await restClient.create(
-        `caches/${cacheId.toString()}`,
+    const additionalHeaders = {
+        "Content-Type": "application/json"
+    };
+    return await httpClient.postJson<any>(
```

Comment:
@bryanmacfarlane 5/5 (last comment for now): This is the other place where postJson<T> is used, but could be more useful if it only *returned* T instead of taking T as input